admin/assets/stylesheets/application.scss

```diff
@@ -416,6 +416,8 @@ table.spacious {
   }
 
   th, td {
+    display: table-cell;
```

Comment:
this line fixes the spacing and aligning problems that appear when a table cell is highlighted (green left border)